modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/intrasharprecordedcell/v0.1.0.json

```diff
@@ -0,0 +1,88 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/experiment/intrasharprecordedcell/v0.1.0/shapes/"
+    }
```

Comment:
I guess it would get really long, but I would be tempted to use *intracellularsharprecordedcell* instead of intrasharprecordedcell

Reply:
I'd be happy with that.